runtime/lua/vim/lsp/diagnostic.lua

```diff
@@ -208,7 +208,7 @@ end
 
 --- @param uri string
 --- @param client_id? integer
---- @param diagnostics vim.Diagnostic[]
+--- @param diagnostics lsp.Diagnostic[]
```

Comment:
I don't think that's correct?

Reply:
@616b2f

Reply:
(It's not correct)

Reply:
Sorry, didn't see this was under the lsp namespace. @goanders?

Reply:
This is used in `on_publish_diagnostics` function below which passes `diagnostics` property from `lsp.PublishDiagnosticsParams` class and there it has `lsp.Diagnostic` type. So it has to be here also `lsp.Diagnostics`.